### ISSUE TITLE
hideDelay not configurable

### ISSUE THREAD

**Issue by adambowles** [tvkhoa/react-tippy#52]
I set `hideDelay={1000}` and the tooltip is unloading instantly on `mouseout` event.

Setting `delay={1000}`, however, will set the show and hide delays to 1000ms

**Comment by pjosh** [comment]
I have the same problem, but on the other side.
I want to set `delay={3000}` to show the tooltip after 3s, but keep the hide instantaneous. It could be amazing if we can configure `delay` & `hideDelay` separately.

**Comment by adambowles** [comment]
That is in fact the same as what I'm experiencing @pjosh

**Comment by gothy** [comment]
Hey folks, how can get the instantaneous hide behavior? :)

Here's my setup for the wrapping component:

```js
<Tooltip
        arrow
        unmountHTMLWhenHide
        position="right"
        size="small"
        animation="none"
        animateFill={false}
        transitionFlip={false}
        delay={0}
        hideDelay={0}
        distance={20}
        duration={0}
        hideDuration={0}
        {...this.props}
      />
```

animation and all possible delays and durations are set to `0`, but tooltip still "hangs around" for 300-500ms after cursor goes away from the target area.

**Comment by mandriv** [comment]
Any update on that issue?
`hideDelay` prop is basically useless right now

**Comment by tbehunin** [comment]
@gothy Yeah, @levindixon's approach worked for me. The key is to **NOT** also add the `animation` attribute. If you include that attribute - _even if it's set to `none`_, it won't immediately show/hide. So, **ONLY** use `duration={0}` and you'll get the affect you want.

**Comment by Kadrian** [comment]
Wow, this should be more obvious! How about no animation by default?

**Comment by NathanVss** [comment]
> @gothy Yeah, @levindixon's approach worked for me. The key is to **NOT** also add the `animation` attribute. If you include that attribute - _even if it's set to `none`_, it won't immediately show/hide. So, **ONLY** use `duration={0}` and you'll get the affect you want.

My hero !
